Model/Cc.php

```diff
@@ -599,12 +599,9 @@ protected function getConsumerInfo($order)
      * @param object $order
      * @return string
      */
-    private function getCallbackUrl($order)
+    private function getNotifyUrl($order)
```

Comment:
oh this is great! thanks! I was good with either Notify or Notification especially since it's only used in this class